cpp/src/c_api/core_number.cpp

```diff
@@ -76,7 +76,7 @@ struct core_number_functor : public cugraph::c_api::abstract_functor {
       rmm::device_uvector<edge_t> core_numbers(graph_view.local_vertex_partition_range_size(),
                                                handle_.get_stream());
 
-      auto degree_type = reinterpret_cast<cugraph::k_core_degree_type_t>(degree_type);
```

Comment:
this does not make any sense, where is degree_type defined ? ... it should be degree_type_

Reply:
yeah this needs the trailing underscore in the function arg variable, as does the other one.